modules/multilanguage/src/main/scala/com/ukg/surge/multilanguage/MultilanguageGatewayServiceImpl.scala

```diff
@@ -42,7 +42,10 @@ class MultilanguageGatewayServiceImpl(aggregateName: String, eventsTopicName: St
     engine
   }
 
-  override def forwardCommand(in: ForwardCommandRequest): Future[ForwardCommandReply] = {
+  private val metric: Metrics = Metrics.globalMetricRegistry
+  private val forwardCommandMetric: Timer = metric.timer(MetricInfo("ForwardCommand", "Forward Command Metric"))
```

Comment:
Same comment here and for the getStateMetric below for the metric name/description

Reply:
Changed